3-sqlite-server/3-2-alter-tasks/server.js

```diff
@@ -0,0 +1,51 @@
+import Database from "better-sqlite3";
+import * as queries from "./queries.js";
```

Comment:
```suggestion
import { TaskQuery } from "./queries.js";
```

* alias import 使う前に named import にした方がいいかも
named import 飛び越えてちゃうとどういう構造で queries を呼べてるかわからなくなりそうなので